helper/tests/helper-tests-14/src/test/java/com/linkedin/avroutil1/compatibility/avro14/CodeTransformationsAvro14Test.java

```diff
@@ -44,7 +44,19 @@ public void testTransformAvro14HugeRecord() throws Exception {
     Schema schema = AvroCompatibilityHelper.parse(avsc);
     String originalCode = runNativeCodegen(schema);
 
-    String transformedCode = CodeTransformations.transformParseCalls(originalCode, AvroVersion.earliest(), AvroVersion.latest());
+    String transformedCode = CodeTransformations.transformParseCalls(originalCode, AvroVersion.AVRO_1_4, AvroVersion.earliest(), AvroVersion.latest());
+
+    Class<?> transformedClass = CompilerUtils.CACHED_COMPILER.loadFromJava(schema.getFullName(), transformedCode);
+    Assert.assertNotNull(transformedClass);
+  }
+
+  @Test
+  public void testTransformAvro14RecordWithMultilineDoc() throws Exception {
```

Comment:
Do we need to duplicate these tests? Can't we parameterize the Avro version and re-use the same code?

Reply:
this test only exists in 2 places - for avro 1.4 and avro 1.5
compared to the annoyance of parameterized tests i'd rather keep it this way.

Reply:
Sure, no worries.